mailbox/api/src/main/java/org/apache/james/mailbox/MailboxListener.java

```diff
@@ -411,10 +412,12 @@ public final int hashCode() {
         private final List<MessageUid> uids;
         private final List<UpdatedFlags> updatedFlags;
 
-        public FlagsUpdated(MailboxSession.SessionId sessionId, User user, MailboxPath path, MailboxId mailboxId, List<MessageUid> uids, List<UpdatedFlags> updatedFlags) {
+        public FlagsUpdated(MailboxSession.SessionId sessionId, User user, MailboxPath path, MailboxId mailboxId, List<UpdatedFlags> updatedFlags) {
```

Comment:
You don't need modifications in EventFactory?

Reply:
it is here https://github.com/linagora/james-project/pull/2055/commits/a071ef5cae186f69ca982af3905e13199d60d378#r242465375